plugins/org.jboss.tools.openshift.ui/META-INF/MANIFEST.MF

```diff
@@ -60,7 +60,7 @@ Export-Package: org.jboss.tools.openshift.internal.ui;x-friends:="org.jboss.tool
  org.jboss.tools.openshift.internal.ui.explorer;x-friends:="org.jboss.tools.openshift.test",
  org.jboss.tools.openshift.internal.ui.handler;x-friends:="org.jboss.tools.openshift.test",
  org.jboss.tools.openshift.internal.ui.job;x-friends:="org.jboss.tools.openshift.test",
- org.jboss.tools.openshift.internal.ui.models;x-friends:="org.jboss.tools.openshift.test",
+ org.jboss.tools.openshift.internal.ui.models,
  org.jboss.tools.openshift.internal.ui.portforwading;x-friends:="org.jboss.tools.openshift.test",
  org.jboss.tools.openshift.internal.ui.property;x-friends:="org.jboss.tools.openshift.test",
  org.jboss.tools.openshift.internal.ui.property.build;x-friends:="org.jboss.tools.openshift.test",
```

Comment:
Package 'org.jboss.tools.openshift.internal.ui.propertytester' does not exist in this plug-in